sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -755,6 +774,14 @@ ProgramManager::getOrCreateKernel(const ContextImplPtr &ContextImpl,
     return Cache.getOrInsertKernel(Program, KernelName);
   };
 
+  if (std::getenv("SYCL_CACHE_DISABLE")) {
```

Comment:
```suggestion
  if (SYCLConfig<SYCL_CACHE_DISABLE>::get())) {
```
And everywhere else in the patch